qa/rpc-tests/p2p-fullblocktest.py

```diff
@@ -629,6 +632,7 @@ def update_block(block_number, new_transactions):
             scriptSig = CScript([sig, redeem_script])
 
             tx.vin[1].scriptSig = scriptSig
+            bloat_tx(tx)
```

Comment:
Wouldn't calling bloat_tx() in tx.rehash() would make this change more pervasive and less error prone? (not to mention the nice side effect of fewer line-changes :) )

Reply:
Then again, having that side effect where it's not expected might make debugging difficult...

Reply:
Ya, realized my mistake when I started poking around at the code some more myself.